components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -16,67 +15,112 @@ import Link from 'next/link';
+        isEliminated ? 'bg-muted' : 'transparent',
+      )}
     >
-      <h4
-        data-testid="league-entry-number"
-        className={cn(
-          'league-entry-number text-xl font-semibold leading-none tracking-tight',
-          isEliminated ? 'opacity-50' : '',
-        )}
+      <section
+        className="league-entry-header flex items-center gap-12"
+        data-testid="league-entry-header"
       >
-        {entryName}
-      </h4>
-      <EntryStatus isEliminated={isEliminated} />
-    </section>
-    <section
-      className="league-entry-footer flex items-center justify-end gap-12"
-      data-testid="league-entry-footer"
-    >
-      {isPickSet && (
-        <img
-          className="league-entry-logo h-12 w-12"
-          data-testid="league-entry-logo"
-          src={teamLogo}
-          alt="teamLogo"
-        />
-      )}
-
-      <div
-        className="league-entry-pick-button-container"
-        data-testid="league-entry-pick-button-container"
+        <h4
+          data-testid="league-entry-number"
+          className={cn(
+            'league-entry-number text-xl font-semibold leading-none tracking-tight',
+            isEliminated ? 'opacity-50' : '',
+          )}
+        >
+          {entryName}
+        </h4>
+        <EntryStatus isEliminated={isEliminated} />
+      </section>
+      <section
+        className="league-entry-footer flex items-center justify-end gap-12"
+        data-testid="league-entry-footer"
       >
-        {!isEliminated && (
-          <Link href={linkUrl} data-testid="league-entry-pick-button-link">
-            <Button
-              className="league-entry-pick-button"
-              data-testid="league-entry-pick-button"
-              label={isPickSet ? 'Change Pick' : 'Make Pick'}
-              variant={isPickSet ? 'secondary' : 'default'}
-            />
-          </Link>
+        {isPickSet && (
+          <img
+            className="league-entry-logo h-12 w-12"
+            data-testid="league-entry-logo"
+            src={teamLogo}
+            alt="teamLogo"
+          />
         )}
-      </div>
-    </section>
-  </div>
-);
+
+        <div
+          className="league-entry-pick-button-container"
+          data-testid="league-entry-pick-button-container"
+        >
+          {!isEliminated && (
+            <LinkCustom
+              aria-disabled={isLockedOut === true ? 'true' : 'false'}
+              className={
+                isLockedOut === true ? 'opacity-50 cursor-not-allowed' : ''
+              }
+              data-testid="league-entry-pick-link"
+              href={linkUrl}
+              onClick={(e: { preventDefault: () => unknown }) =>
+                isLockedOut === true && e.preventDefault()
+              }
+              size={'defaultButton'}
+              variant={isPickSet ? 'secondaryButton' : 'primaryButton'}
```

Comment:
Why are we using `isLockedOut === true` in line 112, but here we're checking for truthy/falsy values?

Reply:
I could not get the functionally working without explicitly checking if `isLockedOut === true`. When I built the `LeagueEntries` component previously I was able to make it work with the truthy/falsy values. 

I'm all for making it more explicit if that is the consensus.

Reply:
@ryandotfurrer @choir27 I think it has to do with how the prop is being brought in and the useEffect is changing the state of isLockedOut.

Reply:
@ryandotfurrer we don't need to bring this in as a prop as the parent component calling it is not doing anything to change that value.

Remove the prop, set the default state to false.  and then see if that allows you to do isLockedOut without the check if it === true.